website/content/docs/provisioners/hcp-sbom.mdx

```diff
@@ -0,0 +1,161 @@
+---
+description: |
+  The `hcp-sbom` Packer provisioner downloads the SBOM file from remote and
+  sends to HCP Packer when build is done.
+page_title: HCP SBOM - Provisioners
+---
+
+<BadgesHeader>
+  <PluginBadge type="official"/>
+</BadgesHeader>
+
+# HCP SBOM Provisioner
+
+Type: `hcp-sbom`
+
+The `hcp-sbom` Packer provisioner downloads the SBOM file from the remote machine
+and sends it to HCP Packer when the build is complete (only if the template is
+HCP-enabled). The SBOM file is automatically removed at the end of the process.
+If the user wants to retain a copy of the SBOM file, they should specify the
+`destination` field in the provisioner.
+
+Currently, we support `CycloneDX` and `SPDX` SBOM formats in `JSON`.
+
+TBA: Add more details about max number of files allowed to download, and
```

Comment:
Is that a TODO for us? We should not publish that I'd think

Reply:
We are not going to publish this but just added it here until we finish the adding other stuff. I am removing this part in my next commit. Will add the PR comment.

Reply:
Sounds good to me!